LaCE-onboarding-guide.md

```diff
@@ -111,7 +111,7 @@ This page will help you solve common queries that you may face while working on
 
 ![Reload-Single-Collection](images/LaCEOnboardingGuide/Reload-Single-collection.png)
 
-  - Load dummy new structures data: This section provides a combination of topics, skills, stories and exploration.
+  - Load dummy new structures data: This section provides a combination of topics, skills, stories and exploration. 
```

Comment:
@Aakash-Jakhmola do you actually need this change (adding a space at the end)? If not then please drop it.